### ISSUE TITLE
Uncaught TypeError: _this.notificationEl.querySelector is not a function

### ISSUE THREAD

**Issue by ofadeyi** [mulesoft/api-workbench#478]
[Enter steps to reproduce:]

1. ...
2. ...

**Atom**: 1.17.0-beta4 x64
**Electron**: 1.3.15
**OS**: Mac OS X 10.12.4
**Thrown From**: [api-workbench](https://github.com/mulesoft/api-workbench) package 0.8.45

### Stack Trace

Uncaught TypeError: _this.notificationEl.querySelector is not a function

```
At /Users/ofadeyi/.atom/packages/api-workbench/node_modules/atom-package-deps/lib/view.js:22

TypeError: _this.notificationEl.querySelector is not a function
    at /packages/api-workbench/node_modules/atom-package-deps/lib/view.js:22:58
```

### Commands

```
```

### Non-Core Packages

```
api-workbench 0.8.45 
```

**Comment by ddenisenko** [comment]
@ofadeyi what are the reproduction steps?

@dreamflyer please check if you can reproduce it on the beta of 1.1.17 Atom

**Comment by dreamflyer** [comment]
@ofadeyi can not to reproduce, exact reproduction sequence is required.

**Comment by LeGaS** [comment]
I just ran into the same issue.

Reproduction steps:
1) Download Atom (1.17.0 at the time of writing)
2) Install api-workbench package
3) Installing dependencies fail when it is installing the linter.

![image](https://cloud.githubusercontent.com/assets/58868/26131455/7933098c-3a9a-11e7-81a7-b9ff49348622.png)

**Comment by rainerh** [comment]
If I install the linter package and its dependencies manually, api-workbench starts without issues.

**Comment by adam-lynch** [comment]
Exact same as @LeGaS and like @rainerh says, manually installing the linter package and restarting fixes it.

**Comment by dreamflyer** [comment]
will be published in v0.8.46.
